test/chi/js/modal.pug

```diff
@@ -0,0 +1,244 @@
+---
+title: Modal
+---
+script(src='/js/chi.js')
+
+// Trigger
```

Comment:
Please remove the comments.
We use comments only in special cases.

Reply:
Fixed in the latest commit